src/test/task.test.js

```diff
@@ -0,0 +1,149 @@
+  test('Remove a completed task in the local storage', () => {
+    output = [
+      { description: 'New Task', index: 1, completed: false },
+    ];
+    tasks = removeCompletedTasks(tasks);
+    expect(tasks).toMatchObject(output);
+    expect(localStorage.getItem('storage')).toMatchObject(tasks);
+  });
+});
+
+describe('Edit a tasks', () => {
+  test('Check If edit task is working', () => {
+    const description = 'The Task has been Edited';
+    const id = 1;
+    output = [
+      { description, index: 1, completed: false },
+    ];
+    edit(tasks, description, id);
+    expect(tasks).toMatchObject(output);
+  });
+});
+
+describe('Update the local storage after Drag/Drop', () => {
+  test('test', () => {
+    addTask(tasks, 'second task');
+    addTask(tasks, 'third task');
+    updateStorage(tasks);
+    output = [
+      { description: 'third task', index: 1, completed: false },
+      { description: 'The Task has been Edited', index: 2, completed: false },
+      { description: 'second task', index: 3, completed: false },
+    ];
+    const list = document.querySelector('#list');
+    list.innerHTML = `<li id="3" class="task" draggable="true">
+                        <div class="description-container">
+                            <input type="checkbox" class="check">
+                            <p class=" description"> third task</p>
+                            <input type="text" id="edit-Description" class="edit-description d-none">
+                        </div>
+                        <i class="fas fa-trash" aria-hidden="true"></i>
+                        <i class="fas fa-edit d-none" aria-hidden="true"></i>
+                     </li>
+                     <li id="1" class="task" draggable="true">
+                        <div class="description-container">
+                            <input type="checkbox" class="check">
+                            <p class=" description"> The Task has been Edited</p>
+                            <input type="text" id="edit-Description" class="edit-description d-none">
+                        </div>
+                        <i class="fas fa-trash" aria-hidden="true"></i>
+                        <i class="fas fa-edit d-none" aria-hidden="true"></i>
+                     </li>
+                     <li id="2" class="task" draggable="true">
+                        <div class="description-container">
+                          <input type="checkbox" class="check">
+                          <p class=" description"> second task</p>
+                          <input type="text" id="edit-Description" class="edit-description d-none">
+                        </div>
+                        <i class="fas fa-trash" aria-hidden="true"></i>
+                        <i class="fas fa-edit d-none" aria-hidden="true"></i>
+                     </li>`;
+    updateAfterDrag(list);
+    tasks = localStorage.getItem('storage');
+    expect(tasks).toMatchObject(output);
+  });
```

Comment:
- [OPTIONAL] You did well writing these tests. Please note that for this task the `add` and `delete` functions were the only ones needed for this project. The following covered tests will be required in a later project(s).